go.mod

```diff
@@ -3,16 +3,16 @@ module sigs.k8s.io/gateway-api
 go 1.15
 
 require (
-	github.com/ahmetb/gen-crd-api-reference-docs v0.2.1-0.20201224172655-df869c1245d4
+	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/lithammer/dedent v1.1.0
 	github.com/russross/blackfriday/v2 v2.1.0 // indirect
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.22.0
-	k8s.io/apimachinery v0.22.0
-	k8s.io/client-go v0.22.0
+	k8s.io/api v0.22.1
+	k8s.io/apimachinery v0.22.1
+	k8s.io/client-go v0.22.1
 	k8s.io/code-generator v0.22.0
-	k8s.io/klog/v2 v2.9.0
-	k8s.io/utils v0.0.0-20210707171843-4b05e18ac7d9
-	sigs.k8s.io/controller-runtime v0.8.3
-	sigs.k8s.io/controller-tools v0.5.0
+	k8s.io/klog/v2 v2.10.0
+	k8s.io/utils v0.0.0-20210820185131-d34e5cb4466e
+	sigs.k8s.io/controller-runtime v0.9.6
+	sigs.k8s.io/controller-tools v0.6.2
```

Comment:
CI is failing for this because of my hack here: https://github.com/kubernetes-sigs/gateway-api/blob/master/hack/update-codegen.sh#L63